### PR TITLE
Update doc and notebooks with -v option

### LINKED ISSUE #127: Update documentation regarding -v option

### PR DESCRIPTION
**Pull request by pierrepo**
Fix #127.

I had to re-rerun Jupyter notebooks to reflect buildH options.

I had some warning messages regarding numpy & MDAnalysis. What was the conclusion regarding the possibility to silent them?

### CODE REVIEW

**Comment by patrickfuchs** [comment]
We decided to leave them as is for now (https://github.com/patrickfuchs/buildH/issues/86). Redirection of *stderr* would silent them but would also hide other relevant errors or warnings... But they pollute the output so much 😞 In the first versions of the notebooks, I removed them by hand hopping in the future they will be fixed by MDAnalysis developers.

**Comment by patrickfuchs** [comment]
Something else, not directly related though, it seems that some tests fail since a few days, which is the case in this PR. Most of the time, it looks like there's a problem with pandoc. But I don't understand, we didn't change anything regarding this. Any idea?

**Comment by HubLot** [comment]
Seems pandoc it's not installed by default now.
I'll look at it

File: docs/notebooks/Notebook_01_buildH_calc_OP.ipynb [hunk 1]
```diff
@@ -280,32 +291,45 @@
...
+      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
+      "  residx = np.zeros_like(criteria[0], dtype=np.int)\n",
+      "/home/pierre/.soft/miniconda3/envs/buildh/lib/python3.9/site-packages/MDAnalysis/core/selection.py:521: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.\n",
+      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
+      "  mask = np.zeros(len(group), dtype=np.bool)\n",
```
Comment (patrickfuchs) [inline docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: Suggestion: delete by hand those #!*$ warnings

Comment (pierrepo) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: The user might not understand why those error messages occur. So it's probably better to let them but adding an explanation that everything is under control.

Comment (patrickfuchs) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: We already tell the user here about the warnings: https://buildh.readthedocs.io/en/latest/usage.html#numpy-warnings
To me that's enough.

Comment (HubLot) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: I would remove those also since it's occurs only for numpy 1.20

Comment (patrickfuchs) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: So @pierrepo, do you agree to remove those warnings?

Comment (pierrepo) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: I would disagree for two reasons:
1. The user might think there is a real issue with buildH if the output in the notebook differs from the one he/she could experience in his/her terminal.
2. It implies to modify manually the json in the notebook that is usually not a good idea.

Could be try updating [MDAnalysis to 1.1.1](https://github.com/MDAnalysis/mdanalysis/releases)? It looks like some Numpy deprecation messages have been handled?

Comment (patrickfuchs) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: > Could we try updating [MDAnalysis to 1.1.1](https://github.com/MDAnalysis/mdanalysis/releases)? It looks like some Numpy deprecation messages have been handled.

Yes, very good idea! Maybe we can create another issue for that, so I can merge this PR which is about the `-v` option?

Comment (pierrepo) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:307]: Yes. We might have to rebuild the notebooks anyway if we upgrade buildH version.

**Pull request merged**